trace/stats.go

```diff
@@ -141,37 +142,30 @@ func (s *PacketCounter) Summary(n int) *PacketCountSummary {
 	}
 }
 
-func topNByTcpPacketCount[T comparable](counts map[T]*PacketCounts, n int) map[T]*PacketCounts {
-	if n <= 0 {
-		return map[T]*PacketCounts{}
-	}
+type pair[T constraints.Ordered] struct {
+	k T
+	v *PacketCounts
+}
 
+func topNByTcpPacketCount[T constraints.Ordered](counts map[T]*PacketCounts, n int) map[T]*PacketCounts {
```

Comment:
Please add a comment to specify expected behaviour. (The case of having ties did occur to me, but I figured you really did want to return them all.)